### PR TITLE
refactor: drop the push-sbom-to-pyxis task

### PR DESCRIPTION
**Pull request by ralphbean**
This change is prompted by https://issues.redhat.com/browse/PSSECMGT-3181

There, manifest tooling was created that pulls the sboms directly from our published images. The pyxis entities are no longer in use.

See corresponding e2e-test change at https://github.com/redhat-appstudio/e2e-tests/pull/1082

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ call Pyxis API). The way to do this is to create a file with mock shell function
 as the commands you want to mock) and inject this file to the beginning of each `script` field in
 the task step that needs mocking.
 
-For reference implementation, check [push-sbom-to-pyxis/tests/](/tasks/push-sbom-to-pyxis/tests/). Here's a break down of how it's done:
+For reference implementation, check [push-sbom-to-pyxis/tests/](https://github.com/redhat-appstudio/release-service-catalog/tree/fb4002711259c167c9815cc32a9ca9a6f5b065ca/tasks/push-sbom-to-pyxis/tests/). Here's a break down of how it's done:
```
Comment (davidmogar) [inline CONTRIBUTING.md]: Why is this changed? We shouldn't map to specific commits as this will go through different environments and keep changing.

Comment (ralphbean) [reply CONTRIBUTING.md]: Oh, because in the change I am _deleting_ the push-sbom-to-pyxis task.

Unfortunately, it was the task that was used as an *example* in CONTRIBUTING.md for how to write a good test.

If I just refer to the directory, it will be blank when users go to visit it.

Alternatively, we could rewrite the docs to use a _different_ task as an example; something other than push-sbom-to-pyxis. But, I wasn't up for that work yet.

Comment (theflockers) [reply CONTRIBUTING.md]: +1. I think the example should reside in an actual _example task_ that we should maintain.

**Comment by scoheb** [comment]
I have **successfully** used this PR content for a push the RH stage using the `rh-push-to-registry-redhat-io` pipeline

See https://github.com/scoheb/release-service-catalog/tree/rhel-bootc-staging

File: CONTRIBUTING.md (hunk 1 above)
Comment (scoheb) [reply CONTRIBUTING.md]: updated text to use an example from another task

File: pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml
Comment (johnbieren) [inline pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml:1]: This file doesn't remove the task

**Review by johnbieren — CHANGES_REQUESTED** [review]
I don't know what `manifest-box` means in the READMEs but the `rh-push-to-external-registry.yaml` file is not correct that is why it is failing CI

File: pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml
Comment (ralphbean) [reply pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml:1]: Sorry - that must've gotten lost in the shuffle. It should be removed now in aa5922e2e36096c0e7c893d444e20635bd3b4a54.

**Review by johnbieren — APPROVED** [review]
